package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
remove this, please

Reply:
Didn't mean to commit this, was a random npm i a while ago.